pages/admin/programs/index.tsx

```diff
@@ -0,0 +1,146 @@
+import {
```

Comment:
Can we make this the `program/index.tsx` page instead? That's what the navbar assumes and usually, we use singulars instead of plural for page routes because it doesn't give any semantic meaning. Ie: program -> all program, program/[id] -> singular program.